File: component-acgnews/src/main/runalone/java/com/rabtman/acgnews/AcgNewsMainActivity.java
package com.rabtman.acgnews;

import android.os.Bundle;
import android.widget.FrameLayout;
import butterknife.BindView;
import butterknife.ButterKnife;
import com.rabtman.acgnews.mvp.ui.fragment.AcgNewsMainFragment;
import com.rabtman.common.base.SimpleActivity;

/**
 * @author Rabtman
 */

public class AcgNewsMainActivity extends SimpleActivity {

  @BindView(R.id.layout_load_fragment)
  FrameLayout layoutLoadFragment;

  @Override
  protected int getLayoutId() {
    return R.layout.activity_runalone;
  }

  @Override
  protected void initData() {
    AcgNewsMainFragment acgNewsMainFragment = new AcgNewsMainFragment();
    loadRootFragment(R.id.layout_load_fragment, acgNewsMainFragment);
  }
}

File: component-acgnews/src/main/runalone/java/com/rabtman/acgnews/AcgNewsMainActivity.kt
package com.rabtman.acgnews

import android.widget.FrameLayout
import butterknife.BindView
import com.rabtman.acgnews.mvp.ui.fragment.AcgNewsMainFragment
import com.rabtman.common.base.SimpleActivity

/**
 * @author Rabtman
 */
class AcgNewsMainActivity : SimpleActivity() {

    @BindView(R2.id.layout_load_fragment)
    lateinit var layoutLoadFragment: FrameLayout
    override fun getLayoutId(): Int {
        return R.layout.activity_runalone
    }

    override fun initData() {
        val acgNewsMainFragment = AcgNewsMainFragment()
        loadRootFragment(R.id.layout_load_fragment, acgNewsMainFragment)
    }
}